freqtrade/freqtradebot.py

```diff
@@ -79,6 +79,9 @@ def __init__(self, config: Dict[str, Any]) -> None:
             self.config.get('edge', {}).get('enabled', False) else None
 
         self.active_pair_whitelist: List[str] = self.config['exchange']['pair_whitelist']
+        if not self.active_pair_whitelist:
+            raise DependencyException('Whitelist is empty.')
```

Comment:
I think you'll want this on line 198(ish) ... it's only called once here on initialization ... never during "real" operations ...
also there, don't raise, but only log it - the dependencyexception we had was cought in maybe_buy ... it'll bubble up too high from `_process()` ...

Reply:
Here it is. A test should be added...